test/unit/node_test.rb

```diff
@@ -201,8 +211,10 @@ class NodeTest < ActiveSupport::TestCase
     assert_not_nil node.tagnames
     assert node.tagnames.first.is_a?(String)
     assert_equal 'test awesome spectrometer activity:spectrometer', node.tagnames.join(' ')
+    assert_equal 'test cool spectrometer activity:spectrometer', node.tagnames.join(' ')
```

Comment:
ah ok -- so we should add cool after "awesome" on the above line instead of adding a new line. Same for the below version too, on 217! Thanks!!! 👍 😄

Reply:
thanks! @jywarren